base_local_planner/include/base_local_planner/local_planner_limits.h

```diff
@@ -79,6 +80,7 @@ class LocalPlannerLimits
       double nacc_lim_theta,
       double nacc_lim_trans,
       double nxy_goal_tolerance,
+      double ninner_xy_goal_tolerance,
```

Comment:
Q: what are these n.... parameters?  sounds weird....   ninner_.....

Reply:
Yeah I'm not a fan of this nomenclature either.... It's the name they gave to the arguments in the constructor... We just use them to initialize the private members.

Reply:
:hankey: